app/src/main/java/com/google/droidjump/GameFragment.java

```diff
@@ -16,29 +16,43 @@
 
 package com.google.droidjump;
 
-import static androidx.navigation.Navigation.findNavController;
+import android.graphics.Point;
 import android.os.Bundle;
+import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.Button;
-
+import java.util.Objects;
 
 public class GameFragment extends Fragment {
 
+    private GameView gameView;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Point screen = new Point();
+        Objects.requireNonNull(getActivity()).getWindowManager().getDefaultDisplay().getSize(
+                screen);
+        gameView = new GameView(getActivity(), screen.x, screen.y, getArguments(), /* isPlaying= */true);
```

Comment:
where do you set these arguments?

Reply:
When you click on any button to navigate to another screen you can pass any arguments you want. In this case, arguments contain the last player level or chosen level from levels screen.